components/StructuredTab/StructuredTab.tsx

```diff
@@ -112,6 +117,18 @@ const TitleContainer = styled.p`
 	font-size: 12px;
 	font-family: ${(props) => props.theme.fonts.extended};
 	text-transform: uppercase;
+	margin: 0;
+	color: ${(props) => props.theme.colors.white};
+`;
+
+const Description = styled.h6`
+	font-size: 8px;
+	color: #757688;
```

Comment:
Wanna use `${props => props.theme.color.gray}`?
Also two `text-align` accidentally got in here